modules/apm/README.md

```diff
@@ -0,0 +1,50 @@
+# APM module
+
+## Usage
+
+```terraform
+provider "newrelic" {
+  account_id    = "123456"
+  api_key       = "NRAK-xxx"
+  admin_api_key = "NRAA-xxx"
+  region        = "US"  # or EU
+}
+
+module "newrelic_alerts" {
+  source      = "github.com/claranet/terraform-newrelic-alerts//modules/apm"
+  environment = "production"
+  policy_name = "APM anomalies detected"
+}
+```
+
+## Requirements
+
+| Name | Version |
+|------|---------|
+| terraform | >= 0.13 |
```

Comment:
Why not `>= 0.12.26`?

The `source` field of the `required_providers` block is backward compatible with terraform 0.12.26+: https://www.terraform.io/docs/configuration/provider-requirements.html#v0-12-compatible-provider-requirements

Reply:
I didn't try, but I don't see why limiting to tf 0.13 in this case 👍